### PR TITLE
Oracles: add to altair and centrifuge runtimes

### LINKED ISSUE #1401: Loans: add oracles to Altair and Centrifuge runtimes
### Description

Currently, oracles are already in a working state in development. We need to port this to Altair and Centrifuge.

### PR DESCRIPTION
**Pull request by lemunozm**
# Description

Fixes #1401

### CODE REVIEW

File: runtime/altair/src/lib.rs [hunk 1]
```diff
@@ -1236,7 +1237,71 @@ impl pallet_block_rewards::Config for Runtime {
...
+	// <https://github.com/open-web3-stack/open-runtime-module-library/issues/920> be merged.
+	#[cfg(feature = "runtime-benchmarks")]
+	type Members = runtime_common::oracle::benchmarks_util::Members;
+	type OnNewData = PriceCollector;
+	type OracleKey = OracleKey;
+	type OracleValue = Rate;
+	type RootOperatorAccountId = RootOperatorOraclePrice;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
+	type WeightInfo = ();
```
Comment (lemunozm) [inline runtime/altair/src/lib.rs:1283]: It scares, but it's the correct way to set the weights in `orml_oracle`: https://github.com/open-web3-stack/open-runtime-module-library/blob/0482bf2b02ac8fea1279da0a34c0900fd369c444/oracle/src/weights.rs#L38

File: runtime/altair/src/lib.rs [hunk 2]
```diff
@@ -1236,7 +1237,71 @@ impl pallet_block_rewards::Config for Runtime {
...
+	type WeightInfo = pallet_membership::weights::SubstrateWeight<Self>;
+}
+
+impl orml_oracle::Config for Runtime {
+	type CombineData = runtime_common::oracle::LastOracleValue;
+	type MaxHasDispatchedSize = MaxHasDispatchedSize;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Members = PriceOracleMembership;
+	// This can be removed once
+	// <https://github.com/open-web3-stack/open-runtime-module-library/issues/920> be merged.
```
Comment (NunoAlexandre) [inline runtime/altair/src/lib.rs:1274]:
```suggestion
	// <https://github.com/open-web3-stack/open-runtime-module-library/issues/920> is merged.
```

File: runtime/altair/src/lib.rs [hunk 3]
```diff
@@ -1236,7 +1237,71 @@ impl pallet_block_rewards::Config for Runtime {
 
 parameter_types! {
 	pub const MaxActiveLoansPerPool: u32 = 300;
+	pub const MaxRateCount: u32 = MaxActiveLoansPerPool::get();
+	pub const MaxCollectionSize: u32 = MaxActiveLoansPerPool::get();
 	pub const MaxWriteOffPolicySize: u32 = 100;
+	pub const MaxPriceOracleMembers: u32 = 5;
+	pub const MaxHasDispatchedSize: u32 = production_or_benchmark!(
+		MaxPriceOracleMembers::get(),
+		// For benchmarking we need a number of members equals to the active loans.
```
Comment (NunoAlexandre) [inline runtime/altair/src/lib.rs:1246]:
```suggestion
		// For benchmarking, we need a number of members equal to the active loans.
```

**Review by NunoAlexandre — APPROVED** [review]
LGTM

File: runtime/altair/src/lib.rs (hunk 3 above)
Comment (lemunozm) [reply runtime/altair/src/lib.rs:1246]: Thanks for the review Nuno!

Comment (NunoAlexandre) [reply runtime/altair/src/lib.rs:1246]: Reviewing your code is a pleasure 😘

**Comment by lemunozm** [comment]
@mustermeiszer could you give your approval/double-check on this? 🙏🏻

File: runtime/altair/src/lib.rs [hunk 4]
```diff
@@ -1236,7 +1237,71 @@ impl pallet_block_rewards::Config for Runtime {
...
 	pub const MaxWriteOffPolicySize: u32 = 100;
+	pub const MaxPriceOracleMembers: u32 = 5;
+	pub const MaxHasDispatchedSize: u32 = production_or_benchmark!(
+		MaxPriceOracleMembers::get(),
+		// For benchmarking we need a number of members equal to the active loans.
+		// The benchmark distintion can be removed once
+		// <https://github.com/open-web3-stack/open-runtime-module-library/issues/920> is merged.
+		MaxActiveLoansPerPool::get()
+	);
+	pub const MaxPoolsWithExternalPrices: u32 = 50;
```
Comment (mustermeiszer) [inline runtime/altair/src/lib.rs:1251]: What is the upper bound and why?

**Review by mustermeiszer — APPROVED** [review]
LGTM! Just a minor question where I am laking understanding.

**Pull request merged**